build/reladomolib.spec

```diff
@@ -51,7 +51,6 @@ central,org.glassfish.jersey.core,jersey-client,2.25,jar,lib/serial,416A00562F25
 central,org.glassfish.jersey.ext,jersey-entity-filtering,2.25,jar,lib/serial,52B5A204A79D22041F4B30C2670E95C2456CBCBD
 central,org.glassfish.jersey.media,jersey-media-json-jackson,2.25,jar,lib/serial,2C99D40047A50FC43C5886545F7F2148C7E4A384
 central,org.glassfish.jersey.media,jersey-media-jaxb,2.25,jar,lib/serial,09AA3EEBA90DCE24F04BF27CD1A4E0D378EC697F
-central,org.glassfish.jersey.bundles.repackaged,jersey-guava,2.25,jar,lib/serial,4439BDDB870B210E40CA2E953813930BE424DC0C
```

Comment:
do the serial tests continue to work without this? try `build reladomo-serial-test-suite`

Reply:
restored